libraries/psibase/native/src/ExecutionContext.cpp

```diff
@@ -51,7 +51,7 @@ namespace psibase
       auto codeHash = sha256(code.pos, code.remaining());
 
       auto account = database.kvGet<CodeRow>(CodeRow::db, codeKey(service));
-      check(account.has_value(), "setCode: unknown service account");
+      check(account.has_value(), "setCode: unknown service account: " + service.str());
```

Comment:
It shouldn't be possible to trigger this.  This `setCode` is only run by the genesis action, which unconditionally creates the `CodeRow` first.

Reply:
I remember getting it when I removed the -sys suffixes, and it was from one of the packages having a postinstall script that had a hex-encoded deprecated service name. This was needed to figure out where to look.

Reply:
That should have caused an error at L196, not here.

Reply:
I'm not in a position to try to reproduce the error to confirm or disconfirm, would you prefer I
1. Add the `service.str()` output to L196, or
2. Replace this change with one at L196

Reply:
I don't really care whether there's a change here. It should only be triggered by changes to native, but it doesn't really hurt. I only really care that if you're adding extra info, it should be added in the place that can be reached by client code.

Reply:
I did option 2